portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** A comment object. */
+public final class Comment {
+
+  private final long id;
+  private final String name;
+  private final String message;
+  private final long timestamp;
+
+  public Comment(long id, String name, String message, long timestamp) {
+    this.id = id;
+    this.name = name;
+    this.message = name + ":     " + message;
```

Comment:
You should store the message as is; if you want to display the name and spaces that should be done in your CSS/HTML.